### PR TITLE
TrimText

### PR DESCRIPTION
**Pull request by VsSekorin**
Apache Commons `StringUtils#trim` OO analog implementation.

### CODE REVIEW

**Comment by 0crat** [comment]
@vssekorin I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/TrimText.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * Text without control characters (char <= 32) from both ends.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Vseslav Sekorin (vssekorin@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ */
+public final class TrimText implements Text {
```
Comment (dusan-rychnovsky) [inline src/main/java/org/cactoos/text/TrimText.java]: Wouldn't `TrimmedText` be a better name?

Comment (VsSekorin) [inline src/main/java/org/cactoos/text/TrimText.java]: Yes, my mistake.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge again

**Pull request merged**